pkgs/by-name/ne/netbird/package.nix

```diff
@@ -73,35 +110,38 @@ buildGoModule (finalAttrs: {
   '';
 
   postInstall =
-    lib.concatStringsSep "\n" (
-      lib.mapAttrsToList (
-        module: binary:
-        ''
-          mv $out/bin/${lib.last (lib.splitString "/" module)} $out/bin/${binary}
+    let
+      builtBinaryName = lib.last (lib.splitString "/" component.module);
+    in
+    # rename binaries
```

Comment:
nit: I think it's obvious what it does. Either include an example or remove comments like these :)